lib/src/code_generator/objc_built_in_functions.dart

```diff
@@ -396,3 +390,37 @@ class ObjCInternalGlobal extends LookUpBinding {
     binding?.addDependencies(dependencies);
   }
 }
+
+/// A wrapper around the objc_msgSend function, or the stret or fpret variants.
+class ObjCMsgSendFunc {
+  final bool isStret;
```

Comment:
Nit: It seems kind of weird we have 3 variants but then a bool.

Can we have an enum, and then use it for setting `originalName` as well?